board.rb

```diff
@@ -6,48 +6,39 @@ class Board
   def initialize(columns, rows)
     @columns = columns
     @rows = rows
-    @board =  self.makeBoard()
+    @board = makeBoard
   end
-
  
   def makeBoard
     grid = Array.new(@columns) 
 
-    for i in 0...grid.length
-      grid[i] = Array.new(@rows)
+    grid.each_with_index do |_column,index|
+      grid[index] = Array.new(@rows)
```

Comment:
Another way to do the same `Array.new(@colums) { Array.new(@rows) }`

Reply:
Fixed, thanks!